kroma-validator/challenger.go

```diff
@@ -623,6 +663,7 @@ func (c *Challenger) CanCreateChallenge(ctx context.Context, outputIndex *big.In
 	cCtx, cCancel := context.WithTimeout(ctx, c.cfg.NetworkTimeout)
```

Comment:
Can you change the comment in L588 to `// call challenger timeout to take challenger's bond away`?